src/main/java/org/broadinstitute/dsm/model/participant/ParticipantWrapper.java

```diff
@@ -94,17 +89,7 @@ private void fetchAndPrepareDataByFilters(DDPInstance ddpInstance, Map<String, S
                     queryBuilder.setParser(parser);
                     boolQueryBuilder.must(queryBuilder.build());
                 }
-                else if (DBConstants.DDP_PARTICIPANT_DATA_ALIAS.equals(source)) {
-                    participantData = new ParticipantDataDao().getParticipantDataByInstanceIdAndFilterQuery(Integer.parseInt(ddpInstance.getDdpInstanceId()), filters.get(source));
-
-                    //if study is AT TODO
-                    if ("atcp".equals(ddpInstance.getName())) {
-                        DefaultValues defaultValues =
-                                new DefaultValues(participantData, esData.getEsParticipants(), ddpInstance, filters.get(source));
-                        participantData = defaultValues.addDefaultValues();
-                    }
-                }
-                else if ("ES".equals(source)){
+                else if (ElasticSearchUtil.ES.equals(source)){
                     //source is not of any study-manager table so it must be ES
```

Comment:
that comment is definitely not true anymore =)

Reply:
actually it is, if source(alias) is not dsm related table aliases like first name, last name which do not come from dsm related tables.